azure/route.sh

```diff
@@ -62,3 +101,28 @@ fn azure_route_table_delete_route(name, group, routetable) {
 						--route-table-name $routetable
 	)
 }
+
+# azure_route_table_route_get_id will return the route ID.
+# `name` is the route name in a route table.
+# `group` is name of resource group.
+# `routetable` is name of route table.
+fn azure_route_table_get_id(name, group, routetable) {
```

Comment:
Ctrl+C,Ctrl+V detected. I think what you mean is `azure_route_table_route_get_id`